src/clique_error.erl

```diff
@@ -82,10 +83,19 @@ format(_Cmd, {error, {nodedown, Node}}) ->
     status(io_lib:format("Target node is down: ~p~n", [Node]));
 format(_Cmd, {error, bad_node}) ->
     status("Invalid node name");
+format(_Cmd, {error, {{badrpc, Reason}, Node}}) ->
+    status(io_lib:format("RPC to node ~p failed for reason: ~p~n", [Node, Reason]));
 format(_Cmd, {error, {conversion, _}}=TypeError) ->
     %% Type-conversion error originating in cuttlefish
     status(cuttlefish_error:xlate(TypeError)).
 
+-spec badrpc_to_error(string() | node(), {badrpc, term()}) -> err().
+badrpc_to_error(Node, {badrpc, nodedown}) ->
+    {error, {nodedown, Node}};
+badrpc_to_error(Node, {badrpc, rpc_process_down}) ->
+    {error, {rpc_process_down, Node}};
+badrpc_to_error(Node, {badrpc, Reason}) ->
+    {error, {{badrpc, Reason}, Node}}.
```

Comment:
Doesn't seem like this needs to partial. Why not factor out the badrpc term: (Node, Reason) -> Error.